src/api/article/models.js

```diff
@@ -0,0 +1,53 @@
+import mongoose, { Schema } from 'mongoose';
+import User from 'api/user';
+
+const ArticleTypeSchema = new Schema({
```

Comment:
If you want can separate models by different files: `article.model.js` & `type.model.js`